charts/ingress-nginx/values.yaml

```diff
@@ -34,6 +34,9 @@ controller:
   # -- Use an existing PSP instead of creating one
   existingPsp: ""
 
+  # -- Enables grpc-port and alpha feature "controlplane" if different than -1
```

Comment:
To be precise, we need it to be greater than 0

Because if you want to listen on port 0, the behavior is non-standard

Reply:
fixed